ksql-common/src/main/java/io/confluent/ksql/metrics/ConsumerCollector.java

```diff
@@ -115,7 +115,7 @@ private String getCounterKey(String topic) {
     // Note: synchronized due to metrics registry not handling concurrent add/check-exists
     // activity in a reliable way
     synchronized (this.metrics) {
-      addSensor(key, "messages-per-sec", new Rate(), sensors, false);
+      addSensor(key, "c-messages-per-sec", new Rate(), sensors, false);
```

Comment:
any reason why we can't call it `consumer-...` would be easier to understand what it is for

Reply:
sure - I used c- because there are a bunch of other metrics in ConsumerCollector named with the same prefix. I'll update those as well.